themes/digital.gov/layouts/partials/core/get_sharetools.html

```diff
@@ -6,21 +6,21 @@
 <section class="share_tools">
   <div class="grid-row">
     <div class="grid-col-4">
-      <a class="twitter" href="http://twitter.com/share?url=https://digital.gov{{ (urls.Parse .Permalink)}}{{$path}}&amp;text={{ .Title }}">
+      <a class="twitter" alt="Twitter share link" href="http://twitter.com/share?url=https://digital.gov{{ (urls.Parse .Permalink)}}{{$path}}&amp;text={{ .Title }}">
```

Comment:
It might be a friendlier experience if we use plain language

```suggestion
      <a class="twitter" aria-label="Share on Twitter" href="http://twitter.com/share?url=https://digital.gov{{ (urls.Parse .Permalink)}}{{$path}}&amp;text={{ .Title }}">
```

Reply:
Updated